jenkins/pipeline.groovy

```diff
@@ -247,6 +264,8 @@ pipeline {
                                         sleep 5
                                     done
 
+                                    sudo yum -y install jq
```

Comment:
Do we need the same for `apt`?

Reply:
I don't think so, we run this pipeline on amazon-linux-2 (micro-amazon worker) which is CentOS 7 based